library/extractor/src/test/java/com/google/android/exoplayer2/extractor/ogg/OggPageHeaderTest.java

```diff
@@ -48,7 +48,7 @@ public void skipToNextPage_success() throws Exception {
                 new byte[] {'O', 'g', 'g', 'S'},
                 TestUtil.buildTestData(20, random)),
```

Comment:
This test fails based on the changes. As we set the limit now to EMPTY_PAGE_HEADER_SIZE which of size 27. So we should use a number greater than 27 here.